amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -3760,3 +3919,100 @@ definitions:
           - SiteCollaborator
           - SiteContributor
           - SiteManager
+  EventPaging:
+    type: object
+    properties:
+      list:
+        type: object
+        properties:
+          pagination:
+            $ref: '#/definitions/Pagination'
+          entries:
+            type: array
+            items:
+              $ref: '#/definitions/EventEntry'
+  EventEntry:
+    required:
+      - entry
+    properties:
+      entry:
+        $ref: '#/definitions/Event'
+  Event:
+    type: object
+    required:
+      - id
+      - name
+      - type
+    properties:
+      id:
+        type: string
+        description: this is the id of the event
+      name:
+        type: string
+        description: This is the unique display label of the event
+      type:
+        type: string
+        description: this is event type
+  EventBodyCreate:
+    type: object
+    required:
+      - name
+      - type
```

Comment:
This isn't required because it has a default.

Reply:
I think this should be kept. It'll help the user when playing around with Swagger UI.  Even though swagger will populate the create body with the default value, a user can simply delete that parameter as it is not required!

Reply:
Kept as it is.

Reply:
Done

Reply:
I meant that swagger UI will show the **asterisk** next to the property. Anyway, unfortunately we are not consistent across our APIs in this matter. 
In general, I agree with not having the default property as required (apart from making the payload simpler, a user may save a tiny amount of bandwidth). Of course, as long as the Service impl adds the appropriate documented value.